pkg/fanal/walker/tar_test.go

```diff
@@ -16,6 +17,9 @@ import (
 )
 
 func TestLayerTar_Walk(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
This test is essential for container image scanning. Is it hard to run tests on Windows?